knative-operator/pkg/controller/knativekafka/knativekafka_controller.go

```diff
@@ -52,7 +52,7 @@ var (
 	role              = mf.Any(mf.ByKind("ClusterRole"), mf.ByKind("Role"))
 	rolebinding       = mf.Any(mf.ByKind("ClusterRoleBinding"), mf.ByKind("RoleBinding"))
 	roleOrRoleBinding = mf.Any(role, rolebinding)
-	KafkaHAComponents = []string{"kafka-ch-controller"}
+	KafkaHAComponents = []string{"kafka-ch-controller", "kafka-controller"}
```

Comment:
Should we add the webhook as well?

Reply:
@pierDipi ah, yeah - I forgot about it.  Added it to the mix

Reply:
Hey @pierDipi @matzew . Is "kafka-ch-controller" supposed to stay there in the list? This component is being removed in 1.23

Reply:
I guess this PR isn't related to that, wasn't it removed on main?

Reply:
removed from 1.23 as well, https://github.com/openshift-knative/serverless-operator/blob/73978d20a6dcbf9f780a39c9756edd7d5bbd61c2/knative-operator/pkg/controller/knativekafka/knativekafka_controller.go#L53

Reply:
Cool. Thanks!